fastq/tenXGenomics.go

```diff
@@ -6,42 +6,33 @@ import (
 	"github.com/vertgenlab/gonomics/fileio"
 )
 
-type LinkedRead struct {
+type TenXPair struct {
 	Fwd *Fastq
 	Rev *Fastq
 	Bx  []dna.Base
 	Umi []dna.Base
 }
 
-func ReadToChanLinked(fileOne string, fileTwo string, tenXg chan<- *LinkedRead) {
+func ReadToChanTenXPair(fileOne string, fileTwo string, barcodeLength int, umiLength int, tenXg chan<- *TenXPair) {
```

Comment:
I am approving as this may be beyond the scope of this pull request, however, from Eric's testing it seems the consensus is that chans sending pointers is generally bad. How do you feel about slowly adjusting this function and the rest of the `ReadToChan` functions in gonomics to passing structs instead of pointers?

Reply:
I can't remember the reason why, but if we are tending in this way, i think the same is true for sending a struct with pointers as data fields.

Reply:
As I clicked enter I started to remember, pointers to structs (`*LinkedRead`) and pointers fields (`*Fwd, *Rev`) will be allocated on the heap by default for the duration of the program. We would only consider using sending a pointer through channels if we need to later modify the struct that it originally pointed to (which is really uncommon).